packages/components/src/components/Snackbar/Snackbar.tsx

```diff
@@ -86,12 +82,18 @@ export type SnackbarProps = Omit<ToastProps, 'data' | 'message'> &
  *
  * ```jsx
  * return (
- *   <SnackbarProvider>
- *     <App />
- *   </SnackbarProvider>
+ *   <SafeAreaProvider
+ *     <SnackbarProvider>
+ *       <App />
+ *     </SnackbarProvider>
+ *   </SafeAreaProvider
  * )
  * ```
  *
+ * **Note:** The snackbar requires [react-native-safe-area-context](https://github.com/th3rdwave/react-native-safe-area-context).
```

Comment:
Need to move `react-native-safe-area-context` from a devDependency to dependency for it to work in consuming apps that wouldn't otherwise have it.